lms/djangoapps/monitoring/scripts/generate_code_owner_mappings.py

```diff
@@ -20,26 +20,26 @@
 # Maps edx-platform installed Django apps to the edx repo that contains
 # the app code. Please add in alphabetical order.
 EDX_REPO_APPS = {
-    'bulk_grades': 'https://github.com/edx/edx-bulk-grades',
+    'bulk_grades': 'https://github.com/openedx/edx-bulk-grades',
     'coaching': 'https://github.com/edx/platform-plugin-coaching',
```

Comment:
This one is private, so I imagine it won't be moving. We should eventually find some other way to handle plugins in this mapping.